src/js/components/TaskListItemComponent.jsx

```diff
@@ -1,51 +1,24 @@
 var classNames = require("classnames");
+var objectPath = require("object-path");
 var React = require("react/addons");
 var Moment = require("moment");
 
+var AppsStore = require("../stores/AppsStore");
 var HealthStatus = require("../constants/HealthStatus");
 var TaskStatus = require("../constants/TaskStatus");
 
-function buildHref(host, port) {
-  return "http://" + host + ":" + port;
-}
-
-function buildTaskAnchors(task) {
-  var taskAnchors;
-  var ports = task.ports;
-  var portsLength = ports.length;
-
-  if (portsLength > 1) {
-    // Linkify each port with the hostname. The port is the text of the
-    // anchor, but the href contains the hostname and port, a full link.
-    taskAnchors = (
+function joinNodes(nodes, separator = ", ") {
+  var lastIndex = nodes.length - 1;
+  return nodes.map((node, i) => {
+    if (lastIndex === i) {
+      separator = null;
+    }
+    return (
       <span className="text-muted">
-        {task.host}:[{ports.map(function (p, index) {
-          return (
-            <span key={p}>
-              <a className="text-muted" href={buildHref(task.host, p)}>
-                {p}
-              </a>
-              {index < portsLength - 1 ? ", " : ""}
-            </span>
-          );
-        })}]
+        {node}{separator}
```

Comment:
@aldipower for some reason my IDE doesn't like this, but eslint is fine with it. I wonder why, though.